src/views/SpecificQuery.js

```diff
@@ -33,10 +33,12 @@ const useStyles = makeStyles((theme) => ({
 
 export const SpecificQuery = () => {
     if (!localStorage.getItem('zmt-token')) navigate('/');
+    const location = useLocation()
+    const params = new URLSearchParams(location.search)
     const classes = useStyles()
     const [status, setStatus] = useState('none')
-    const [newAlias, setNewAlias] = useState()
-    const [newSqlStr, setNewSqlStr] = useState()
+    const [newAlias, setNewAlias] = useState(params.get('alias') ? decodeURIComponent(params.get('alias')) : "");
```

Comment:
semicolon...

Reply:
added semicolons to everything in that section. seems like the other pages/tabs (like `Server.js`) all have semicolons...